MODULE.bazel

```
@@ -1,3 +1,135 @@
+#===============================================================================
+# Copyright 2025 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
 module(name = "onedal")
 
+bazel_dep(name = "platforms", version = "0.0.5")
```

```suggestion
bazel_dep(name = "platforms", version = "0.0.10")
```
May resolve `WARNING: For repository 'platforms', the root module requires module version platforms@0.0.5, but got platforms@0.0.10 in the resolved dependency graph. Please update the version in your MODULE.bazel or set --check_direct_dependencies=off`. Unless there is a specific reason to set to 0.0.5.